packages/realm/src/ClassHelpers.ts

```diff
@@ -26,17 +26,14 @@ import {
   binding,
 } from "./internal";
 
-type BindingObjectSchema = binding.Realm["schema"][0];
-
 type ObjectWrapper = (obj: binding.Obj) => (RealmObject & DefaultObject) | null;
 
 /**
  * @internal
  */
 export type ClassHelpers = {
   constructor: RealmObjectConstructor;
-  // TODO: Use a different type, once exposed by the binding
-  objectSchema: BindingObjectSchema;
+  objectSchema: binding.ObjectSchema;
```

Comment:
Are these completely equivalent? It might not be a problem, but I vaguely remember it having optional properties for the properties which have defaults in Core, making it suitable as a type for something passed into Core whereas it's not as ideal for something read from Core.

Reply:
I believe they should be equivalent. The type of  `binding.Realm["schema"]` is `readonly binding.ObjectSchema[]` from:
```ts
type BindingObjectSchema = binding.Realm["schema"][0];
```
There's usually the `_Relaxed` convention when deviating from what's needed for Core, but do you think there are types that we use that are not using that convention?